Description: put the 'examples' folder into /usr/share/doc/pd-* so that
 lintian does not complain about images being in lib directory, then link to
 it in the library so that the Pd Help Browser can find it.
Author: Hans-Christoph Steiner <hans@eds.org>
Forwarded: not-needed
Reviewed-By: Hans-Christoph Steiner <hans@eds.org>
Last-Update: <2010-11-05>

--- pd-moonlib-0.2.orig/Makefile
+++ pd-moonlib-0.2/Makefile
@@ -206,9 +206,9 @@
 
 install-examples:
 	test -z "$(strip $(EXAMPLES))" || \
-		$(INSTALL_DIR) $(DESTDIR)$(objectsdir)/$(LIBRARY_NAME)/examples && \
+		$(INSTALL_DIR) $(DESTDIR)$(prefix)/share/pd-$(LIBRARY_NAME)/examples && \
 		for file in $(EXAMPLES); do \
-			$(INSTALL_DATA) examples/$$file $(DESTDIR)$(objectsdir)/$(LIBRARY_NAME)/examples; \
+			$(INSTALL_DATA) examples/$$file $(DESTDIR)$(prefix)/share/pd-$(LIBRARY_NAME)/examples; \
 		done
 
 install-manual:
